lib/client/renderer.js

```diff
@@ -53,7 +53,7 @@ function init (client, d3) {
 
   // get the desired opacity for context chart based on the brush extent
   renderer.highlightBrushPoints = function highlightBrushPoints (data) {
-    if (data.mills >= chart().brush.extent()[0].getTime() && data.mills <= chart().brush.extent()[1].getTime()) {
+    if (client.latestSGV && data.mills >= chart().brush.extent()[0].getTime() && data.mills <= chart().brush.extent()[1].getTime()) {
```

Comment:
At the very first start there is no measured values and the user can see errors in console.